src/lib.rs

```diff
@@ -161,6 +161,22 @@ impl fmt::Debug for Location {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { (self as &dyn fmt::Display).fmt(f) }
 }
 
+/// Interface variable component.
+#[derive(PartialEq, Eq, Hash, Default, Clone, Copy)]
+pub struct Component(u32);
```

Comment:
Would it be more handy to have `Location` and `Component` merged into something like `InterfaceLocation` (`ilocation`), just like `DescriptorBinding` (`desc_bind`)?